include/libtorrent/torrent_handle.hpp

```diff
@@ -694,7 +694,7 @@ TORRENT_IPV6_NAMESPACE_END
 		//		if (!h.is_valid()) continue;
 		//		torrent_status s = h.status();
 		//		if (!s.has_metadata) continue;
-		//		if (!s.need_save_resume_data()) continue;
+		//		if (!h.need_save_resume_data()) continue;
```

Comment:
I believe this function is deprecated, in favor of the torrent flag (on torrent_status::flags)

Reply:
it doesn't seem to be deprecated, but it's more efficient to check `s.flags & torrent_flags_t::need_save_resume_data` instead